examples/basic/annotations/legends_item_visibility.py

```diff
@@ -1,3 +1,11 @@
+''' Marker and line plots that demonstrate manual control of legend visibility of individual items
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.circle, bokeh.plotting.figure.line
+    :refs: :ref:`ug_basic_annotations_hiding-legend-items`
```

Comment:
this reference anchor does not exist in the sphinx docs source under `docs/bokeh`

There is a general `ug_basic_annotations` that could be used, @tcmetzger do you know if there are any better references that could be used?

Reply:
Do you mean `hiding-legend-items` @bryevdv? 
If yes, then I took it from this [link](https://docs.bokeh.org/en/latest/docs/user_guide/basic/annotations.html#hiding-legend-items)

Reply:
@bryevdv okay sir. I'm on it.